website/docs/guides/custom-cicd-pipelines.md

```diff
@@ -509,7 +515,7 @@ Additionally, you’ll see the job in the run history of dbt Cloud. It should be
 
 If your git provider is not one with a native integration with dbt Cloud, but you still want to take advantage of CI builds, you've come to the right spot! With just a bit of work it's possible to setup a job that will run a dbt Cloud job when a pull request (PR) is created.
 
-:::info Run on PR
+:::tip Run on PR
 
 If your git provider has a native integration with dbt Cloud, you can take advantage of the setup instructions [here](/docs/deploy/ci-jobs).
```

Comment:
for consistency and parallel construction, new callout on line 61 follows this ^ format